style.css

```diff
@@ -42,17 +42,59 @@ header {
   line-height: 133%;
 }
 
+.change-logo {
+  display: none;
+}
+
 .menu {
   margin-right: 1.125rem;
-  width: 18px;
-  height: 12px;
   color: inherit;
+  transition: 0.7s ease-in-out;
+}
+
+.change-menu {
+  width: 100vw;
+  position: relative;
+  left: 85%;
+  transition: 0.5s ease-in-out;
+  font-size: 1.2rem;
+}
+
+.change-menu-color {
+  color: #6f6c6b;
 }
 
 .open_menu {
   display: none;
 }
 
+.change-open_menu {
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  margin-left: 4px;
+  color: #fff;
+  width: 88%;
+  height: 95vh;
+}
+
+.change-open_menu li {
+  padding: 0 1em;
+  margin-left: 26px;
+  margin-top: 20px;
+  border-bottom: 1px solid #6f6c6b;
+}
+
+.change-open_menu a {
+  color: #fff5e1;
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.8rem;
+  line-height: 138%;
+}
+
```

Comment:
As the width increases, kindly make sure that the dimensions and font-size of the menu links (Portfolio, About, Contact) also increases in order to achieve a more responsive website. 

### Your menu at screen width 727px

![Screenshot from 2023-01-31 13-33-27](https://user-images.githubusercontent.com/65233860/215774954-5083a663-a96a-4be3-9237-57b52f1fcbde.png)

### The figma design 

![Screenshot from 2023-01-31 13-37-13](https://user-images.githubusercontent.com/65233860/215775043-7a293051-dd3b-4db9-a1db-133e60b2b19a.png)